src/components/pages/Settings/SettingsLayout.tsx

```diff
@@ -0,0 +1,87 @@
+import { BrandLogo } from '@src/components/icons';
+import { NavBar } from '@src/components/layout';
+import Footer from '@src/components/layout/Footer/Footer';
+import { NetworkIcon } from 'lucide-react';
+import { Link, Outlet, useLocation } from 'react-router-dom';
+
+import './styles.css';
+
+const settingsRoutes = [
+  {
+    name: 'ArNS Registry',
+    path: 'arns',
+    icon: <BrandLogo width={'20px'} height={'20px'} fill={'inherit'} />,
+  },
+  {
+    name: 'Network',
+    path: 'network',
+    icon: <NetworkIcon width={'20px'} height={'20px'} fill={'none'} />,
+  },
+];
+
+function SettingsLayout() {
+  const location = useLocation();
+
+  return (
+    <div className="flex flex-col w-full h-screen box-border">
+      <div className="bg-foreground">
+        <NavBar />
+      </div>
+      <div className="flex flex-col w-full h-full px-[100px] py-[30px]">
+        <div className="flex flex-col w-full rounded-t-xl bg-metallic-grey border-x-[1px] border-t-[1px] border-dark-grey p-2">
+          <h1
+            className="flex flex-row text-white text-xl pl-3 "
+            style={{ gap: '15px' }}
+          >
+            <Link
+              to={'/settings'}
+              className={
+                location?.pathname?.split('/')?.at(-1) == 'settings'
```

Comment:
I'd refactor this location check to a variable as it's being repeated multiple times below. 

Sidenote: I haven't been using it, but Tailwind supports data-attributes which might work here: https://tailwindcss.com/docs/hover-focus-and-other-states#data-attributes

Reply:
I refactored - but did not use the data attribute. In this particular case I think the JSX approach is cleaner.

If I had a non binary value that would require a multiline JSX case then I would probably use the data attribute though. Pretty cool that tailwind allows that.